cmd/submit.go

```diff
@@ -75,10 +123,10 @@ func runSubmit(cfg config.Config, flags *pflag.FlagSet, args []string) error {
 
         %s
 
-		`
-				return fmt.Errorf(msg, arg)
+        `
```

Comment:
IIRC, there are inconsistencies with formatting related to template string newlines starting with tabs vs spaces. I'm guessing in many of the template strings, newlines start with a space as an intentional style of output. In my editor (using vim-go) `gofmt` runs on save and `gofmt` uses tabs for indentation so any time one of the existing template strings gets yanked and pasted, the newlines that started with a space become a tab. Making it identical requires manually replacing the tab with the space again.

Reply:
I also use vim-go and run `gofmt` on save. I didn't think that `gofmt` changed literal strings, though, for exactly this reason.

I do know that I was intentional about the indentation that I put in, but I don't know if others made other choices.

I'd like to make template string changes in a different PR (if at all).

Reply:
Let me double check that there is actually something to this that isn't dependent on my editor config. At the time, I remember that `gofmt` seemed to be editing the string literals but it's possible that I was confused. It could be that the act of pasting was what was replacing spaces with tabs. I'll report back.